docs/nightly/en/user-guide/operations/region-failover.md

```diff
@@ -0,0 +1,60 @@
+# Region Failover
+
+Region Failover provides the ability to recover regions from crashed Datanodes without losing data.
+
+:::warning Warning
+This feature is only available on GreptimeDB running on cluster mode and
+
+- Using Kafka WAL
+- Using [shared storage](/user-guide/operations/configuration.md#storage-options) (e.g., AWS S3)
+
+:::
+
+## The Recovery Time of Region Failover
+
+:::tip Note
+
+In best practices, the number of topics/partitions supported by a Kafka cluster is limited (exceeding this number can degrade Kafka cluster performance). Therefore, we allow multiple regions to share a single topic as the WAL.
+:::
+
+The Recovery Time of Region Failover depends on:
+
+- number of regions per Topic.
+- the Kafka cluster read throughput performance.
+
+### The read amplification
+
+However, the cost of multiple regions to share a single topic is read amplification during replaying WAL.
+
+For example, configure 128 topics for [metasrv](/user-guide/operations/configuration.md#metasrv-only-configuration), and if the whole cluster holds 1024 regions (physical regions), every 8 regions will share one topic.
+
+![Read Amplification](/remote-wal-read-amplification.png)
+
+<p style="text-align: center;"><b>(Figure1: recovery Region 0 need to read 7 times the redundant data)</b></p>
+
+A simple model to estimate the read amplification factor:
+
+- For a single topic, the amplification factor is 1 + 2 + ... + 7 = 28 times.
+- When recovering 100 regions (requiring about 12 topics), the amplification factor is approximately 28 \* 12 = 336 times.
+
+| Number of regions per Topic | Number of topics required for 100 Regions | Single topic read amplification factor | Total reading amplification factor | Replay size (GB) |
+| --------------------------- | ----------------------------------------- | -------------------------------------- | ---------------------------------- | ---------------- |
+| 1                           | 100                                       | 0                                      | 0                                  | 0.5              |
+| 2                           | 50                                        | 1                                      | 50                                 | 25.5             |
+| 4                           | 25                                        | 6                                      | 150                                | 75.5             |
+| 8                           | 13                                        | 28                                     | 364                                | 182.5            |
+| 16                          | 7                                         | 120                                    | 840                                | 420.5            |
+
+**If the Kafka cluster can provide 300MB/s read throughput, recovering 100 regions requires approximately 10 minutes (182.5GB/0.3GB = 10m).**
+
+### Examples
```

Comment:
hmmm...the above content is an example, and here is the Examples, it is hard to understand the outline logic